alfred-telemetry-solr/README.md

```diff
@@ -1,79 +1,232 @@
+The jar file of the module, together with all micrometer dependencies need to be added to solr's classpath and solr core need to be configured to use the handler and the writer. See examples in integration tests.
 
-* enableCoreStats
-* enableFTSMetrics
-* enableTrackerMetrics
-* enableJmxMetrics
-    * enableJmxMetricsOS
-    * enableJmxMetricsMemory
-    * enableJmxMetricsClassLoading
-    * enableJmxMetricsGC
-    * enableJmxMetricsThreading
-    * enableJmxMetricsThreadPool
-    * enableJmxMetricsRequests
-    * enableJmxMetricsSessions
-    * enableJmxMetricsSolr
 
-## How to build
+# Supported monitoring systems
+
+At the moment only Prometheus is supported as a monitoring system.
+
+# Supported metrics
+
+At the moment it is not possible to configure which metrics should be included.
+
+## Jvm metrics
+
+JVM metrics binding provides several jvm metrics. 
+At the moment following metrics are included:
+
+* JvmGcMetrics
+* JvmMemoryMetrics
+* JvmThreadMetrics
+* ClassLoaderMetrics
+
+See metrics provided by these modules [here](https://github.com/micrometer-metrics/micrometer/tree/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm).
+
+Note: at the moment GC metrics are not working fully.
+
+## System metrics
+
+System metrics binding provides several system metrics.
+At the moment following metrics are included:
+
+* UptimeMetrics
+* ProcessorMetrics
+* FileDescriptorMetrics
+
+See metrics provided by these modules [here](https://github.com/micrometer-metrics/micrometer/tree/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system).
+
+## Process metrics
+
+Process metrics binding provides several process metrics.
+At the moment following metrics are included:
+
+* ProcessMemoryMetrics
+* ProcessThreadMetrics
+
+See metrics provided by these modules [here](https://github.com/mweirauch/micrometer-jvm-extras/tree/master/src/main/java/io/github/mweirauch/micrometer/jvm/extras).
+
+## Tomcat metrics
+
+Tomcat metrics binding provides several tomcat-related metrics.
+At the moment following metrics are included:
+
+* TomcatMetrics
+* TomcatBeansMetrics (custom code, implemented because session-related metrics from default implementation did not work)
+
+See metrics provided by TomcatMetrics [here](https://github.com/micrometer-metrics/micrometer/tree/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/tomcat).
+
+
+## Custom metrics
+
+Following custom metrics have been implemented.
+
+### Solr core stats metrics
+
+At the moment following metrics are included, for each core tracked by solr:
+
+| Name                                                       |
+| -----------------------------------------------------------|
+| alfresco.nodes{core=\<core\>,feature="Nodes"}                |
+| alfresco.nodes{core=\<core\>,feature="Unindexed Nodes"}      |
+| alfresco.nodes{core=\<core\>,feature="Error Nodes"}          |
```

Comment:
```suggestion
| alfresco.nodes{core=\<core\>,feature="Errors"}          |
```

Similarly, nodes is already in the metric name

Reply:
Will replace with “Indexed”.